did/pkh.go

```diff
@@ -0,0 +1,163 @@
+package did
+
+import (
+	"fmt"
+	"github.com/TBD54566975/ssi-sdk/cryptosuite"
+	"github.com/TBD54566975/ssi-sdk/schema"
+	"github.com/goccy/go-json"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"regexp"
+	"strings"
+)
+
+type (
+	DIDPKH  string
+	Network string
+)
+
+const (
+	// DIDPKHPrefix did:pkh prefix
+	DIDPKHPrefix = "did:pkh"
+)
+
+const (
+	Bitcoin  Network = "Bitcoin"
+	Ethereum Network = "Ethereum"
+	Polygon  Network = "Polygon"
+)
+
+var didPkhNetworkPrefixMap = map[Network][]string{
+	Bitcoin:  {"bip122:000000000019d6689c085ae165831e93", "EcdsaSecp256k1RecoveryMethod2020"},
+	Ethereum: {"eip155:1", "EcdsaSecp256k1RecoveryMethod2020"},
+	Polygon:  {"eip155:137", "EcdsaSecp256k1RecoveryMethod2020"},
+}
+
+// The following context should be manually inserted into each DID Document. This will likely change
+// over time as new verification methods are supported, and general-purpose methods are specified.
+var KnownDIDPKHContext, _ = schema.GetKnownSchema("did-pkh-context.json")
```

Comment:
is this context useful outside of this package - does it need to be exported?

2nd - consider making a const for the context